### DISCUSSION TITLE
Dictionary transforms with different custom readers for image and labels

### DISCUSSION THREAD

**Discussion by ikatechis** [Project-MONAI/tutorials#1121] [Q&A]
Hello everyone,

I would like to read 3d cbct scans with a custom dicom (.dcm) reader for images and custom npz reader for annotations.
Is it possible to use two different readers (and two different LoadImage instances) to load the labels and images and then use dictionary transforms for the subsequent transforms?
What would be the best way to do this?
Thank you!

**Comment by KumoLiu** [comment]
Hi, @ikatechis, you can do something like this:
```
LoadImaged(keys="img", reader="ITKReader"),
LoadImaged(keys="label", reader="NumpyReader"),
Flipd(keys=['img', 'label']),
.......
```
Thanks!

**Comment by ikatechis** [comment]
@KumoLiu Thanks for your reply, I didn't realize I could chain the loading like this. Thanks!
